src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
I actually have a discussion about this, because `renderArea()` actually resets the caching state (as I thought it should)